src/ng-select/themes/bootstrap4.theme.scss

```diff
@@ -0,0 +1,381 @@
+          }
+      }
+      &.ng-select-top {
+          bottom: 100%;
+          border-top-right-radius: $dropdown-border-radius;
+          border-top-left-radius: $dropdown-border-radius;
+          margin-bottom: -1px;
+          .ng-dropdown-panel-items {
+              .ng-option {
+                  &:first-child {
+                      border-top-right-radius: $dropdown-border-radius;
+                      border-top-left-radius: $dropdown-border-radius;
+                  }
+              }
+          }
+      }
+
+
+      .ng-dropdown-header {
+          border-bottom: 1px solid $ng-select-border;
+          padding: 5px 7px;
+      }
+      .ng-dropdown-footer {
+          border-top: 1px solid $ng-select-border;
+          padding: 5px 7px;
+      }
+      .ng-dropdown-panel-items {
+          margin-bottom: 1px;
+          .ng-optgroup {
+              user-select: none;
+              padding: 8px 10px;
+              font-weight: 500;
+              color: rgba(0, 0, 0, 0.54);
+              cursor: pointer;
+              &.ng-option-disabled {
+                  cursor: default;
+              }
+              &.ng-option-marked {
+                  background-color: $ng-select-marked;
+              }
+              &.ng-option-selected, &.ng-option-selected.ng-option-marked {
+                  color: $dropdown-link-active-color;
+                  background-color: $ng-select-selected;
+              }
+          }
+          .ng-option {
+              background-color: $ng-select-bg;
+              padding: $input-padding-y $input-padding-x;
+
+              &.ng-option-selected, &.ng-option-selected.ng-option-marked {
+                  color: $dropdown-link-active-color;
+                  background-color: $ng-select-selected;
+              }
+              &.ng-option-marked {
+                  color: $dropdown-link-active-color;
+                  background-color: $ng-select-marked;
+              }
+              &.ng-option-disabled {
+                  color: $dropdown-link-disabled-color;
+              }
+              &.ng-option-child {
+                  padding-left: 22px;
+                  @include rtl {
+                      padding-right: 22px;
+                      padding-left: 0;
+                  }
+              }
+              .ng-tag-label {
+                  font-size: 80%;
+                  font-weight: 400;
+                  padding-right: 5px;
+                  @include rtl {
+                      padding-left: 5px;
+                      padding-right: 0;
+                  }
+              }
+          }
+      }
+
+      @include rtl {
+          direction: rtl;
+          text-align: right;
+      }
+  }
+
+  .form-control-sm .ng-dropdown-panel .ng-dropdown-panel-items .ng-option {
+    padding: $input-padding-y-sm $input-padding-x-sm;
+  }
+
+  .form-control-lg .ng-dropdown-panel .ng-dropdown-panel-items .ng-option {
+    padding: $input-padding-y-lg $input-padding-x-lg;
+  }
```

Comment:
That will affect bootstrap elements as well. I'm not sure if that is what we want.